src/Translator/doc/index.rst

```diff
@@ -65,12 +65,28 @@ After installing the bundle, the following file should be created, thanks to the
 Usage
 -----
 
-When warming up the Symfony cache, all of your translations will be dumped as JavaScript into the ``var/translations/`` directory.
+When warming up the Symfony cache, your translations will be dumped as JavaScript into the ``var/translations/`` directory.
 For a better developer experience, TypeScript types definitions are also generated aside those JavaScript files.
 
 Then, you will be able to import those JavaScript translations in your assets.
 Don't worry about your final bundle size, only the translations you use will be included in your final bundle, thanks to the `tree shaking <https://webpack.js.org/guides/tree-shaking/>`_.
 
+Restrict the translation domains
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, all your translations will be exported. You can restrict the dumped messages by either
+including or excluding translation domains in your ``config/packages/ux_translator.yaml`` file:
+
+.. code-block:: yaml
+
+    ux_translator:
+        # By listing the domains you want to include:
+        included_domains: ['messages', 'admin']
+
+        # Or by excluding the domains you don't want:
+        excluded_domains: ['validators', 'security']
```

Comment:
What about using the same style monolog uses for channel config:

```suggestion
    ux_translator:
        domains: ~    # Include all the domains

        domains: foo  # Include only domain 'foo'
        domains: '!foo' # Include all domains, except 'foo'

        domains: [foo, bar]   # Include only domains 'foo' and 'bar'
        domains: ['!foo', '!bar'] # Include all domains, except 'foo' and 'bar'
```

Reply:
So this would throw a invalid config exception ? 

```yaml
        domains: ['!foo', 'bar'] 
```

Reply:
I guess just mimic what monolog does in this case.

Reply:
Thanks for the suggestion, I've pushed the changes.